odl/util/graphics.py

```diff
@@ -31,6 +31,36 @@
 __all__ = ('show_discrete_data',)
 
 
+def _safe_minmax(values):
+    """Calculate min and max of array with guards for nan and inf."""
+    # Nan guarded min and max
+    minval = np.nanmin(values)
+    maxval = np.nanmax(values)
+
+    # Get something sensible in case of infinite values
+    if minval == -np.inf:
+        minval = np.nanmin(values[values > -np.inf])
+    if maxval == np.inf:
+        maxval = np.nanmin(values[values < np.inf])
```

Comment:
Isfinite also skips `NaN` values, so can change to `np.min` will do that instead.